### ISSUE TITLE
Prov edit icons only appear for first entity

### ISSUE THREAD

**Issue by gothub** [NCEAS/metacatui#1224]
When in edit mode, the prov icons should display for all entities. Currently only the first set of icons is displaying for the first entity:

<img width="1387" alt="Screen Shot 2020-01-06 at 9 19 40 AM" src="https://user-images.githubusercontent.com/6412946/71835295-02cce000-3066-11ea-8167-e21c595de75a.png">

It appears that an error is occurring for the citation display that is preventing any additional prov edit icons from displaying:

```
CitationView.js?v=2.8.0.1204:31 Uncaught TypeError: Cannot read property 'getType' of null
    at n.initialize (CitationView.js?v=2.8.0.1204:31)
    at n.e.View (backbone.js:1224)
    at new n (backbone.js:1884)
    at n.createNode (ProvChartView.js?v=2.8.0.1204:497)
    at n.<anonymous> (ProvChartView.js?v=2.8.0.1204:163)
    at underscore.js:73
    at Function.m.each.m.forEach (underscore.js:153)
    at n.render (ProvChartView.js?v=2.8.0.1204:129)
    at MetadataView.js?v=2.8.0.1204:1380
    at Function.m.each.m.forEach (underscore.js:153)
```

**Comment by laurenwalker** [comment]
I believe this was fixed in commit 5705c962ac7b18251e7a1e9d6d1d57b412eeccd1 and included in MetacatUI 2.8.2

If you let me know which deployment you saw this error, I will update it. It looks like it is still on 2.8.0.

**Comment by gothub** [comment]
@laurenwalker - thanks for the quick reply -  this was from current arcticdata.io - here is the link: https://arcticdata.io/catalog/view/doi:10.18739/A2MP4VN5F
